WebRoot/css/zm.css

```diff
@@ -1716,6 +1750,14 @@ INPUT[type="text"].search_input-expanded:focus {
 	line-height:normal;
 }
 
+.EditorDragBdr {
+	border-width: 0px !important;
```

Comment:
Our editor is embedded inside tinymce container div.
And, tinymce container div have inline style with some border value assosiated .
So, we are overriding the inline style with !important, here as we need to show the dotted border.
This is however a class that is applied at run time.
When a file enters a drag area and removed when dropped or on drag leave event.
It will not affect the cascading of the elements in this scenario.